From fd5cdf9c39bb7b88ce988f8e7e4514c467e3cde2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Yilli=C3=A9?= <yilliee@protonmail.com>
Date: Sat, 1 Jan 2022 20:46:56 +0500
Subject: [PATCH 1/2] Initialize arrow.mk using lineage.mk as a reference

---
 arrow.mk | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 arrow.mk

diff --git a/arrow.mk b/arrow.mk
new file mode 100644
index 0000000..1c2aadd
--- /dev/null
+++ b/arrow.mk
@@ -0,0 +1,11 @@
+$(call inherit-product, vendor/arrow/config/common.mk)
+$(call inherit-product, vendor/arrow/config/BoardConfigArrow.mk)
+$(call inherit-product, device/arrow/sepolicy/common/sepolicy.mk)
+
+TARGET_NO_KERNEL_OVERRIDE := true
+SELINUX_IGNORE_NEVERALLOWS := true
+
+TARGET_USES_PREBUILT_VENDOR_SEPOLICY := true
+TARGET_HAS_FUSEBLK_SEPOLICY_ON_VENDOR := true
+
+TARGET_FACE_UNLOCK_SUPPORTED := true
-- 
2.34.1

